sdl_android/src/main/java/com/smartdevicelink/streaming/StreamPacketizer.java

```diff
@@ -14,7 +14,14 @@
 
 	private Thread t = null;
 
-	private final static int BUFF_READ_SIZE = 1024;
+
+	private final static int TLS_MAX_RECORD_SIZE = 16384;
+	private final static int TLS_RECORD_HEADER_SIZE = 5;

```

Comment:
@askirk should be 5, looks like I had a type in the issue.  I updated the issue with the proper value. (I also added an issue to the protocol spec with more details) Thanks!